frontend-runitup/src/Components/Feed/FeedPage.jsx

```diff
@@ -61,6 +61,16 @@ const FeedPage = () => {
     );
   };
 
+  const handleCommentAdded = (postId) => {
+    setPosts((prevPosts) =>
+      prevPosts.map((post) =>
+        post.id === postId
+          ? { ...post, commentCount: post.commentCount + 1 }
+          : post
+      )
+    );
+  };
```

Comment:
if you foresee there ever being a large number of posts in this state that you have to iterate through, it might make sense to maintain the `posts` react state in a map/dictionary so you don't have to iterate through each post just to update the comment count for one specific post

Reply:
Mmm okay! I really like this idea!